templates/rhdm73-authoring-ha.yaml

```diff
@@ -292,10 +292,15 @@ parameters:
   description: Maven mirror to use by Decision Central and KIE server.
   name: MAVEN_MIRROR_URL
   required: false
+- displayName: Maven mirror of
+  description: Maven mirror configuration for KIE server.
+  name: MAVEN_MIRROR_OF
+  value: "external:*,!repo-rhdmcentr"
+  required: false
 - displayName: Maven repository ID
-  description: The id to use for the maven repository, if set. Default is generated randomly.
+  description: "The id to use for the maven repository. If set, it can be excluded from the optionally configured mirror by adding it to MAVEN_MIRROR_OF. For example: external:*,!repo-rhdmcentr,!repo-custom. If MAVEN_MIRROR_URL is set but MAVEN_MIRROR_ID is not set, an id will be generated randomly, but won't be usable in MAVEN_MIRROR_OF."
   name: MAVEN_REPO_ID
-  example: my-repo-id
+  example: repo-custom
```

Comment:
We wanted to create a value: external-repo-id here instead of example: my-repo-id